core/forms.py

```diff
@@ -143,3 +143,11 @@ class Meta:
         fields = "__all__"
 
     updating_user_field_name = "uploader"
+
+class AccountForm(ModelForm):
```

Comment:
Personally I'd rather see this particular form placed in `user_interaction/account_pages/forms.py` as this form will only be used in this section and thus is less relevant as part of the core. It's just a personal preference though